README.md

```diff
@@ -1,72 +1,13 @@
 # Mva-meldingen
 
-Documentation for mva-meldingen
+If you visit this github prosject in order to manual test a submission of VAT-report with the Phyton script. 
```

Comment:
If you are visiting this github project in order to manually test a submission of the VAT-report with the Python script,

please visit...